js/screens/EditProfile/EditProfileContainer.js

```diff
@@ -1,32 +1,169 @@
 import React, { Component } from "react";
+import { View } from "react-native";
+import ImagePicker from "react-native-image-picker";
+
+import { connect } from "react-redux";
+import { fetchUser } from "../../redux/modules/user";
 import PropTypes from "prop-types";
+
 import EditProfile from "./EditProfile";
-import { View, Text } from "react-native";
+import Loading from "../../components/Loading/";
+
+import { updateProfileOnEdit } from "../../redux/modules/user";
+
+import { headerBarStyle } from "../../config/styles";
 import { styles } from "./styles";
 
 class EditProfileContainer extends Component {
+  static propTypes = {
+    navigation: PropTypes.object.isRequired
+  };
+  static navigationOptions = ({ navigation }) => ({
+    title: "EDIT PROFILE",
+    ...headerBarStyle(navigation, true)
+  });
   constructor(props) {
     super(props);
+    const {
+      fullname,
+      program,
+      schoolName,
+      aboutMe,
+      workstyle,
+      profilePhoto,
+      chips
+    } = this.props.navigation.state.params;
     this.state = {
-      user: ""
+      fullname,
+      program,
+      schoolName,
+      aboutMe,
+      profilePhoto,
+      chips,
+      workstyle,
+      loading: true
     };
   }
 
-  render() {
-    const { loading } = this.props;
+  componentDidMount() {
+    this.props.dispatch(fetchUser());
+  }
+
+  updateProfilePhoto = () => {
+    const options = {
+      title: "Select Photo",
+      allowsEditing: true,
+      maxHeight: 400,
+      maxWidth: 400,
+      cameraType: "front"
+    };
+    ImagePicker.showImagePicker(options, response => {
```

Comment:
I am pretty sure this file had been changed. Can you do rebase, remove all conflicts and check this file again?